### PR TITLE
CCTRI-2510

### PR DESCRIPTION
**Pull request by oshynk**
- Sumo Client restructuring due to Judgements and Verdicts
- Judgements implementation
- Unit tests

### CODE REVIEW

File: code/api/mapping.py [hunk 1]
```diff
@@ -85,3 +88,89 @@ def extract_sighting(self, message):
...
         return sighting
+
+    def _judgement(self, cs_data):
+        judgement = {
+            **self._disposition(cs_data),
+            'confidence': 'High',
+            'id': self._transient_id(cs_data),
+            'observables': [self.observable],
+            'priority': 85,
+            'schema_version': '1.1.6',
```
Comment (TarasMaliuchenko) [inline code/api/mapping.py:99]: Is it possible here to  prevent code duplication for fields which are the same for multiple CTIM objects?

Comment (oshynk) [reply code/api/mapping.py:99]: I thought about it, but for example Verdicts will not have schema_version and other fields common for Sightings and Judgements. So not sure if it is consistent to have defaults only for Sightings and Judgements.

Comment (TarasMaliuchenko) [reply code/api/mapping.py:99]: Just think about it when you will have time for refactoring

**Review by TarasMaliuchenko — APPROVED** [review]

File: code/api/mapping.py (hunk 1 above)
Comment (oshynk) [reply code/api/mapping.py:99]: Okay, I'll reconsider it while implementing Verdicts

File: code/api/client.py [hunk 2]
```diff
@@ -54,9 +55,43 @@ def _request(self, path, method='GET', body=None,
 
         raise CriticalSumoLogicResponseError(response)
 
-    def get_data(self, observable):
-        search_id = self._create_search(observable)
+    def get_messages(self, observable):
+        search_type = 'Sumo Logic'
```
Comment (AnastasiiaRozlyvan) [inline code/api/client.py:59]: I think it's better to move things like this into constants, as you use it also in `mapping.py`

**Review by ytsek — APPROVED** [review]

File: code/api/client.py (hunk 2 above)
Comment (oshynk) [reply code/api/client.py:59]: It's not the same as the project name actually. It's just improvisation to somehow differ warning messages for two different searches, so at some point this text may be different from those in mapping.py

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Pull request merged**